### DISCUSSION TITLE
Private Commits do not show in my github profile.

### DISCUSSION THREAD

**Discussion by RafaelCENG** [community/community#56665] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello everyone. For a reason my commits do not show in my profile. Im committing in a private repo as collaborator. My PRs are showing as commits. I have my github account connected. Any tips? Couldn't find anything in the forums.

**Comment by RafaelCENG** [comment]
Already checked. When I create a PR as a contributor to the repository the commit is there. But when I push (in branches not main) the commit exists only in the repo and not on my profile

**Reply by airtower-luna** [reply, marked as answer]
That answers your question, only commits on the default branch of a repository will be shown as contributions. Once you merge, they should show up.

See: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#commits

**Comment by nirav-gajera** [comment]


**Reply by RafaelCENG** [reply]
1. I have the correct authorship associated with my GH account
2. I'm pushing with no problems
3. Can't review commit settings since I'm not the owner of the repo.
![image](https://github.com/community/community/assets/45431116/ca39ac5d-ac97-498c-aca1-55462f8ada15)

**Comment by ocman-nazir-briet** [comment]
you need to set them visible manually on your profile page, there will be an option to include them.

**Comment by ocman-nazir-briet** [comment]
![git](https://github.com/community/community/assets/49003689/77317d28-8922-4374-b62b-28f848288064)
you can follow this to include them

**Reply by RafaelCENG** [reply]
Done that. One other guy answered my question thanks.

**Reply by amiraadev** [reply]
thank you , that was helpful

**Comment by Vana12330** [comment]
> ### Select Topic Area
> 
> Question
> 
> ### Body
> 
> Hello everyone. For a reason my commits do not show in my profile. Im committing in a private repo as collaborator. My PRs are showing as commits. I have my github account connected. Any tips? Couldn't find anything in the forums.

**Comment by Evilsoot** [comment]
I don't understand why my account link is not public like when I share my link github.com/<my username> people get 404 error this happens with github.com/Evilsoot it's like the account is only visible to me
